pmd-core/src/main/java/net/sourceforge/pmd/lang/metrics/MetricKeyUtil.java

```diff
@@ -1,68 +0,0 @@
-/**
- * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
- */
-
-package net.sourceforge.pmd.lang.metrics;
-
-import java.util.Objects;
-
-import net.sourceforge.pmd.lang.ast.Node;
-
-/**
- * Holds the key creation method until we move it to the MetricKey interface.
- *
- * @author Clément Fournier
- * @since 6.0.0
- */
-public final class MetricKeyUtil {
```

Comment:
Should we mark this as deprecated on master? It seems, it is only used by tests right now...